code/modules/mob/living/simple_animal/aquatic.dm

```diff
@@ -0,0 +1,37 @@
+//For things that swim and don't do much else.
+/mob/living/simple_animal/aquatic
+	name = "aquatic animal"
+	desc = "You shouldn't be seeing this."
+	icon = 'icons/mob/npc/fish.dmi'
+	icon_state = "fish"
+	item_state = "fish"
+	icon_living = "fish"
+	icon_dead = "fish_dead"
+	icon_rest = "fish_rest"
+	meat_type = /obj/item/reagent_containers/food/snacks/fish/fishfillet
+	organ_names = list("snout", "body", "tail fin", "left pectoral fin", "right pectoral fin", "dorsal fin")
+	response_help = "brushes"
+	response_disarm = "attempts to push"
+	response_harm = "injures"
+
+	var/global/list/suitable_turf_types = list(
+		/turf/simulated/floor/beach/water,
+		/turf/simulated/floor/beach/coastline,
+		/turf/simulated/floor/holofloor/beach/water,
+		/turf/simulated/floor/holofloor/beach/coastline,
```

Comment:
regular fish should preferably only be able to swim in actual water.
consider making a holo subtype of fish that can swim in these turfs, if the intended use is in, for example, holodeck templates.